app/Repositories/Repo.php

```diff
@@ -31,40 +32,43 @@ public static function getRecords(string $table, array $columns=[] , array $filt
         return $records;
     }
 
-    public static function insertRecord(string $table, array $data ):bool
+    public static function insertRecord(string $table, array $data): bool
     {
         return DB::table($table)->insert($data);
     }
 
-    public static function updateRecord(string $table, array $data):bool
+    public static function updateRecord(string $table, int $id, array $data): bool
     {
-        return DB::table($table)->update($data);
+        return DB::table($table)->where('id', $id)->update($data);
     }
 
-    public static function valid(Builder $query):Builder{
-        return $query->where(function (Builder $q){
-            $q ->where('since', '<=', Carbon::createFromFormat('Y-m-d H:i:s', Carbon::now('Asia/Tehran')))
+    public static function valid(Builder $query): Builder
```

Comment:
No use case in project! please remove it.

Reply:
done